java/client/src/org/openqa/selenium/remote/CapabilityType.java

```diff
@@ -47,6 +47,7 @@
   String ELEMENT_SCROLL_BEHAVIOR = "elementScrollBehavior";
   String HAS_TOUCHSCREEN = "hasTouchScreen";
   String OVERLAPPING_CHECK_DISABLED = "overlappingCheckDisabled";
+  String ENABLE_DOWNLOADING = "chromium:enableDownloading";
```

Comment:
The name `enableDownloading` implies this is a boolean capability. How about `downloadDir`?

Reply:
Thank you for review. I believe that "downloadDir" is not so appropriate in this case because it sounds like specifying download directory rather than applying download behavior for specific path. It could cause some misunderstanding as to why we introduce this capability. What about AllowHeadlessDownloadTo?
From one hand, it is boolean, from the other hand, it specify the downloading directory. There are no ways to allow downloading without specifying directory for downloading. There are no way to specify directory for downloading without allowing downloading.

Reply:
It's not specific to headless, so how about `allowDownloadsTo`?